rs/bitcoin/ckbtc/minter/src/state/eventlog.rs

```diff
@@ -204,6 +205,23 @@ mod event {
     }
 }
 
+#[derive(Clone, Eq, PartialEq, Debug, Deserialize, Serialize, candid::CandidType)]
+pub struct Event {
+    /// The canister time at which the minter generated this event.
+    pub timestamp: Option<u64>,
+    /// The event type.
+    pub payload: EventType,
+}
+
+impl Event {
+    pub fn from_event_type(payload: EventType) -> Event {
```

Comment:
nit: this can be implemented via the `From` trait.

Reply:
Good point! Done.